scargo/commands/check.py

```diff
@@ -186,15 +186,25 @@ def check(self) -> None:
         super().check()
 
     def check_file(self, file_path: Path) -> CheckResult:
+        copyrights = self.copyright_desc.split("\n")[:-1]
+        slice_present = 0
+        copyright_present = False
         with open(file_path, encoding="utf-8") as file:
             for line in file.readlines():
-                if self.copyright_desc in line:
-                    return CheckResult(problems_found=0)
-                if "copyright" in line.lower():
-                    logger.warning(
-                        "Incorrect and not excluded copyright in %s", file_path
-                    )
-                    return CheckResult(problems_found=1, fix=False)
+                for copyrights_slice in copyrights:
+                    if copyrights_slice in line:
```

Comment:
Maybe you could improve this check? Note that `in` may accept lines that are different than in the defined copyright text.
For example:
```python
copyright_line = "Copyright 2023"
copyright_line in "This is not Copyright 20234"  # True
```